interfaces/IComposition.h

```diff
@@ -57,7 +57,7 @@ namespace Exchange {
         static uint32_t WidthFromResolution(const ScreenResolution resolution);
         static uint32_t HeightFromResolution(const ScreenResolution resolution);
 
-        static ScreenResolution ResolutionFromHeightWidth(const uint32_t height, const uint32_t width);
+        static ScreenResolution ResolutionFromWidthHeightRefresh(const uint32_t height, const uint32_t width, const uint32_t refresh);
```

Comment:
Can we make referesh an optional parameter, for 2 reasons:
1) Backwards compatibility
2) Usability, I think a lot of consumers of this method are not really interested in the assoicated refresh rate and would than prefer the "highest" refresh rate. So by passing in a default parameter of '0' , whihc is an invalid refresh rate, the implementation can than return the highest available framerate.